static/library/map_component.ts

```diff
@@ -146,16 +162,58 @@ export class DatacommonsMapComponent extends LitElement {
   }
 
   render(): HTMLElement {
-    const place = this.parentPlace || this.place || this.placeDcid;
-    const variable = this.variable || this.statVarDcid;
-    const childPlaceType = this.childPlaceType || this.enclosedPlaceType;
+    let dataSpec = [];
+    if (!_.isEmpty(this.parentPlaces) && !_.isEmpty(this.childPlaceTypes)) {
+      this.parentPlaces?.forEach((placeDcid, index) => {
+        const enclosedPlaceType =
```

Comment:
since the parentPlaces and childPlaceTypes indices have to match, maybe just break `if index > childPlaceTypes.length - 1`

Reply:
I've chosen to leave as is and add an explanatory comment instead. Leaving this allows us to do something like 
```
parentPlaces="country/CAN country/USA country/MEX"
childPlaceTypes="AdministrativeArea1"
```
to plot AA1s for all 3 countries without needing to repeat "AdministrativeArea1" 3 times.

Reply:
Updated comments!